### PR TITLE
setting the body parser limits to 10mb. default value is 100kbb

### LINKED ISSUE #151: 413 Request Entity Too Large error
### Description

When I send a large request to the emulator I get a 413 error (coming from expressjs) with this message:
  expected: 979352,
  length: 979352,
  limit: 102400,
  message: 'request entity too large',
  status: 413,
  statusCode: 413,

but the size limit of a entity in CloudFunctions is 10MB, which should be 10485760 instead of 102400

### Steps to reproduce

send any request to a function with a payload bigger than 100 kb.

Thanks!

### PR DESCRIPTION
**Pull request by aminy**
reference: https://github.com/expressjs/body-parser#limit

Fixes #151 (it's a good idea to open an issue first for discussion)

- [x] - `npm test` succeeds
- [x] - Code coverage does not decrease (if any source code was changed)
- [ ] - If applicable, PR includes appropriate documentation changes

### CODE REVIEW

**Comment by aminy** [comment]
I signed it!

File: src/supervisor/worker.js [hunk 1]
```diff
@@ -69,14 +69,30 @@ function main () {
     }
 
     const app = express();
+    
+    const requestLimit = '1024mb';
```
Comment (divyenduz) [inline src/supervisor/worker.js]: Shouldn't this be '10mb'?

**Comment by jmdobry** [comment]
Nope, I'm matching the settings in production

**Comment by aminy** [comment]
:+1:

**Pull request merged**